blaise_survey_manager/users/functions.py

```diff
@@ -107,3 +110,11 @@ def encrypt_string(unencrypted_string):
         return encrypted_string
     except Exception as ex:
         log.error("Error encrypting string: " + str(ex))
+
+
+def is_manager(role: str) -> bool:
+    # yes, the below roles are hard-coded. See:
+    # https://collaborate2.ons.gov.uk/jira/browse/LU-7570?focusedCommentId=151122&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-151122
+    managerial_roles = ["Field_Manager", "Field_Ops_Manager", "TO_Manager", "TO_Planner", "BDSS", "DST_Tech"]
```

Comment:
DST account does not log
```suggestion
    managerial_roles = ["Field_Manager", "Field_Ops_Manager", "TO_Manager", "TO_Planner", "BDSS", "DST_TECH"]
```